src/components/Navigator/NavigatorCard.vue

```diff
@@ -294,6 +299,7 @@ export default {
     hasFilter({ debouncedFilter, selectedTags }) {
       return Boolean(debouncedFilter.length || selectedTags.length);
     },
+    isDesktop: ({ breakpoint }) => breakpoint === BreakpointName.large,
```

Comment:
Super small thing, but how about just `isLargeBreakpoint` since you can have large breakpoints on non-desktop devices?